clinica/pipelines/t1_freesurfer/t1_freesurfer_cli.py

```diff
@@ -22,36 +23,45 @@ def define_options(self):
                                   help='Path to the BIDS directory.')
         clinica_comp.add_argument("caps_directory",
                                   help='Path to the CAPS directory.')
+
         # Optional arguments (e.g. FWHM)
         optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
         optional.add_argument("-raa", "--recon_all_args",
-                              help='Additional flags for recon-all command line (default: --recon_all_args "-qcache")')
+                              metavar='flag(s)', type=str, default="-qcache",
+                              help='Additional flags for recon-all command line '
+                                   '(default: --recon_all_args="%(default)s")')
+
         # Clinica standard arguments (e.g. --n_procs)
         clinica_opt = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_OPTIONAL'])
         clinica_opt.add_argument("-tsv", "--subjects_sessions_tsv",
                                  help='TSV file containing a list of subjects with their sessions.')
         clinica_opt.add_argument("-wd", "--working_directory",
                                  help='Temporary directory to store pipelines intermediate results')
         clinica_opt.add_argument("-np", "--n_procs",
-                                 metavar=('N'), type=int,
+                                 metavar='N', type=int,
                                  help='Number of cores used to run in parallel')
 
     def run_command(self, args):
         """Run the pipeline with defined args."""
         import os
-        import datetime
         from colorama import Fore
+        from tempfile import mkdtemp
+        from .t1_freesurfer_pipeline import T1FreeSurfer
         from clinica.utils.exceptions import ClinicaException
         from clinica.utils.stream import cprint
-        from .t1_freesurfer_pipeline import T1FreeSurfer
-        from tempfile import mkdtemp
+        from clinica.utils.ux import print_end_pipeline
 
         pipeline = T1FreeSurfer(
             bids_directory=self.absolute_path(args.bids_directory),
             caps_directory=self.absolute_path(args.caps_directory),
             tsv_file=self.absolute_path(args.subjects_sessions_tsv)
         )
 
+        if "-dontrun" in args.recon_all_args.split(' '):
+            cprint('%s[Warning] Found -dontrun flag for FreeSurfer recon-all. '
+                   'Please note that this will not run the segmentation.%s' %
+                   (Fore.YELLOW, Fore.RESET))
+
         pipeline.parameters = {
             'recon_all_args': args.recon_all_args or '-qcache'
```

Comment:
> User can't specify that he doesn't want to use any parameter ?

If this line enforces the use of `-qcache`, this is good. Otherwise, `statistics-surface` will not work with outputs of `t1-freesurfer`.